components/LocationComparisonSection/LocationComparisonWrapper.tsx

```diff
@@ -15,6 +15,8 @@ interface ComponentProps {
   countryCode: string;
   options: SpotlightOptions;
   onFilterChanged: (options: SpotlightOptions) => void;
+  onRemoveChart: (index: number) => void;
+  index: number;
```

Comment:
I need u to implement this without passing the index as a prop ... this component doesn't need to know about indexes, not even in the `onRemoveChart` prop